eng/pipelines/templates/steps/build-test.yml

```diff
@@ -82,6 +82,36 @@ steps:
     inputs:
       BuildDropPath: $(Build.ArtifactStagingDirectory)
 
+  - task: Powershell@2
+    inputs:
+      filePath: $(Build.SourcesDirectory)/eng/common/scripts/Save-Package-Properties.ps1
+      arguments: >
+        -ServiceDirectory ${{parameters.ServiceDirectory}}
+        -OutDirectory $(Build.ArtifactStagingDirectory)/PackageInfo
+      pwsh: true
+      workingDirectory: $(Pipeline.Workspace)
+    displayName: Dump Package properties
+    condition: succeeded()
+
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+  
+  - task: Powershell@2
+    inputs:
+      filePath: $(Build.SourcesDirectory)/eng/scripts/Create-ApiReview.ps1
+      arguments: >
+        -ServiceDirectory ${{parameters.ServiceDirectory}}
+        -OutPath $(Build.ArtifactStagingDirectory)
+        -ApiviewUri "$(azuresdk-apiview-uri)"
```

Comment:
@praveenkuttappan do you know if all pipelines already have access to this or if I need to do a pipeline gen update to propagate this variable?

Reply:
It's already part of our core variable group that gets linked to each pipeline created by pipeline generator.